sys/src/lib9p/file.c

```diff
@@ -72,12 +72,12 @@ freefile(File *f)
 		free(fl);
 	}
 
-	free(f->name);
-	free(f->uid);
-	free(f->gid);
-	free(f->muid);
+	free(f->dir.name);
+	free(f->dir.uid);
+	free(f->dir.gid);
+	free(f->dir.muid);
 	qlock(&filelk);
-	assert(f->ref == 0);
+	assert(f->ref.ref == 0);
```

Comment:
this is fine, be sure not to change this ref b/c there is already ref (not Ref) in that struct.